src/WebApi/Controllers/LeaderboardController.cs

```diff
@@ -19,11 +18,15 @@ public class LeaderboardController : BaseController
     /// <response code="200">Ok.</response>
     [HttpGet("leaderboard")]
     [ResponseCache(Duration = 1 * 60 * 1)] // 1 minutes
-    public Task<ActionResult<Result<IList<CharacterPublicViewModel>>>> GetLeaderboard([FromQuery] Region? region)
+    public Task<ActionResult<Result<IList<CharacterPublicViewModel>>>> GetLeaderboard(
+        [FromQuery] Region? region,
+        [FromQuery] CharacterClass? characterClass
+    )
```

Comment:
this solves the stylecop warning
```suggestion
    public Task<ActionResult<Result<IList<CharacterPublicViewModel>>>> GetLeaderboard(
        [FromQuery] Region? region,
        [FromQuery] CharacterClass? characterClass)
```